### PR TITLE
Sils/vet

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

File: bears/tests/go/test_files/vet_bad.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package main
+func main() {
+    return 1 || 1
+}
```
Comment (rubdos) [inline bears/tests/go/test_files/vet_bad.go]: A newline missing here.

**Comment by sils1297** [comment]
ack 451e999 901ef5b

**Comment by sils1297** [comment]
reack  f9a324d

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**